x-pack/plugins/maps/public/classes/sources/geojson_file_source/geojson_file.test.ts

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { GeoJsonFileSource } from './geojson_file_source';
+import { BoundsFilters } from '../vector_source';
+import { FIELD_ORIGIN } from '../../../../common/constants';
+
+describe('GeoJsonFileSource', () => {
+  describe('getName', () => {
+    it('should get bounds from feature collection', async () => {
```

Comment:
test message does not convey what test is doing. How about changing to "should get source name"?